notebooks/byod.py

```diff
@@ -214,13 +179,34 @@ def create_cram_crai_table(table: str, listing: Iterable[str]):
     gs://my-workspace-bucket/my-crams/NWD3.cram
     gs://my-workspace-bucket/my-crams/NWD3.crai
     ```
-    would produce the table
+    would produce a table that looks like this in Terra.
+
+    | sample_id | cramLocation       | craiLocation      |
+    | --------- | -----------------  | ----------------- |
+    | NWD1      | NWD1.cram          | NWD1.crai         |
+    | NWD2      | NWD2.cram          | NWD2.crai         |
+    | NWD3      | NWD3.cram          | NWD3.crai         |
+
+    However, the actual raw data generated will be more like this:
+
+    | entity:sample_id | cramLocation                                         | craiLocation                                |
+    | ---------        | ---------------------------------------------------  | ------------------------------------------- |
+    | NWD1             | gs://my-workspace-bucket/my-crams/NWD1.cram          | gs://my-workspace-bucket/my-crams/NWD1.crai |
+    | NWD2             | gs://my-workspace-bucket/my-crams/NWD2.cram          | gs://my-workspace-bucket/my-crams/NWD2.crai |
+    | NWD3             | gs://my-workspace-bucket/my-crams/NWD3.cram          | gs://my-workspace-bucket/my-crams/NWD3.crai |
+    """
+
+with callysto.Cell("python"):
+    listing = [key for key in gs.list_bucket("my-crams")]
+    create_cram_crai_table("my-table-name", listing)
+
+with callysto.Cell("markdown"):
+    """
+    Now, go check the data section of your workspace and you should a data table with the name you have given it, and that table should act as a directory of your files.
+
+    If you set the name of your table to a table that already exists, the old table will not be overwritten, but the new table won't be created either. Terra tables cannot be updated, they must be deleted and remade.
```

Comment:
I don't think this is true. You can check this yourself using the methods `upload_data_table`, `upload_rows`, or `upload_columns` defined in this notebook.

Reply:
Interesting... it definitely did work like this when I wrote my own functions, so I assumed it was the same for yours as both rely on Firecloud. I'll double check with some tests like you suggested, but I wouldn't be surprised if this was updated to match the behavior you're expecting because it was definitely frustrating to have to rename the output notebook every time you ran it. Thanks for catching this.